File: Python Fundamentals/Exercises/Exercise_4/qu_2.py
num_1 = int(input())
num_2 = int(input())
num_3 = int(input())


def sum_numbers(a, b):
    sum_nums = a + b
    return sum_nums


def subtract(a, b):
    diff_nums = a - b
    return diff_nums


def add_and_subtract(a, b, c):
    result = subtract(sum_numbers(a, b), c)
    return result


print(add_and_subtract(num_1, num_2, num_3))

File: JS Front-End/03-.JS-Front-End-Functions-and-Statements/Exercises/qu2.js
function sumAndSubstract(num1, num2, num3) {
    let result = sum(num1, num2) - num3;

    function sum(a, b) {
        return a + b;
    }

    console.log(result);
}

// sumAndSubstract(23, 6, 10);
// sumAndSubstract(1, 17, 30);
// sumAndSubstract(42, 58, 100);